clients/src/main/resources/common/message/MetadataResponse.json

```diff
@@ -77,10 +77,10 @@
         { "name": "OfflineReplicas", "type": "[]int32", "versions": "5+", "ignorable": true,
           "about": "The set of offline replicas of this partition." }
       ]},
-      { "name": "TopicAuthorizedOperations", "type": "int32", "versions": "8+",
+      { "name": "TopicAuthorizedOperations", "type": "int32", "versions": "8+", "default": "-2147483648",
```

Comment:
we should also update DescribeGroupsResponse.json.

Reply:
@omkreddy Colin is away for a week or so, so maybe you can address this when you have a chance. This is a blocker for 2.3.